keanu-project/src/main/java/io/improbable/keanu/vertices/VertexDictionary.java

```diff
@@ -0,0 +1,17 @@
+package io.improbable.keanu.vertices;
+
+import java.util.Map;
+
+import io.improbable.keanu.tensor.Tensor;
+
+public interface VertexDictionary {
+    <V extends Vertex<? extends Tensor<?>>> V get(VertexLabel label);
```

Comment:
nit: sorry I need whitespace here to be able to sleep soundly at night

Reply:
done